src/main/java/ch/njol/skript/Skript.java

```diff
@@ -187,6 +187,7 @@ public final class Skript extends JavaPlugin implements Listener {
 
 	private static org.skriptlang.skript.@UnknownNullability Skript skript = null;
 	private static org.skriptlang.skript.@UnknownNullability Skript unmodifiableSkript = null;
+	private static final ReflectUtils REFLECT_UTILS = ReflectUtils.getInstance();
```

Comment:
this nullifies all the benefits of making the utils an instances
the instance should be new for each onEnable call, so set it there or provide some sort of reset method.